### ISSUE TITLE
RuntimeError: conflicting chdir during another chdir block

### ISSUE THREAD

**Issue by lekemula** [lzap/gem-ripper-tags#4]
--- ERROR REPORT TEMPLATE -------------------------------------------------------

```
RuntimeError: conflicting chdir during another chdir block
  /Users/lekemula/.rvm/gems/ruby-3.3.1/gems/gem-ripper-tags-1.0.6/lib/rubygems/commands/ripper_tags_command.rb:48:in `chdir'
          /Users/lekemula/.rvm/gems/ruby-3.3.1/gems/gem-ripper-tags-1.0.6/lib/rubygems/commands/ripper_tags_command.rb:48:in `index'
          /Users/lekemula/.rvm/gems/ruby-3.3.1/gems/gem-ripper-tags-1.0.6/lib/rubygems_plugin.rb:12:in `block in <top (required)>'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path/installer.rb:43:in `block in run_hooks'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path/installer.rb:42:in `each'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path/installer.rb:42:in `run_hooks'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path/installer.rb:34:in `post_install'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path.rb:245:in `generate_bin'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/source/path.rb:89:in `install'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/installer/gem_installer.rb:54:in `install'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/installer/gem_installer.rb:16:in `install_from_spec'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/installer/parallel_installer.rb:132:in `do_install'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/installer/parallel_installer.rb:123:in `block in worker_pool'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/worker.rb:62:in `apply_func'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/worker.rb:57:in `block in process_queue'
          <internal:kernel>:187:in `loop'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/worker.rb:54:in `process_queue'
          /Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/3.3.0/bundler/worker.rb:90:in `block (2 levels) in create_threads'

```

## Environment

```
Bundler             2.5.9
  Platforms         ruby, arm64-darwin-22
Ruby                3.3.1p55 (2024-04-23 revision c56cd86388092faec079981f779f140717020d58) [arm64-darwin-22]
  Full Path         /Users/lekemula/.rvm/rubies/ruby-3.3.1/bin/ruby
  Config Dir        /Users/lekemula/.rvm/rubies/ruby-3.3.1/etc
RubyGems            3.5.9
  Gem Home          /Users/lekemula/.rvm/gems/ruby-3.3.1
  Gem Path          /Users/lekemula/.rvm/gems/ruby-3.3.1:/Users/lekemula/.rvm/rubies/ruby-3.3.1/lib/ruby/gems/3.3.0
  User Home         /Users/lekemula
  User Path         /Users/lekemula/.gem/ruby/3.3.0
  Bin Dir           /Users/lekemula/.rvm/gems/ruby-3.3.1/bin
OpenSSL
  Compiled          OpenSSL 1.1.1w  11 Sep 2023
  Loaded            OpenSSL 1.1.1w  11 Sep 2023
  Cert File         /opt/homebrew/etc/openssl@1.1/cert.pem
  Cert Dir          /opt/homebrew/etc/openssl@1.1/certs
Tools
  Git               2.39.2 (Apple Git-143)
  RVM               1.29.12-next (master)
  rbenv             rbenv 1.2.0
  chruby            not installed
  rubygems-bundler  (1.4.5)
```

## Bundler Build Metadata

```
Built At          2024-08-13
Git SHA           unknown
Released Version  false
```

## Bundler settings

```
jobs
  Set via BUNDLE_JOBS: 10
```

**Comment by lzap** [comment]
https://bugs.ruby-lang.org/issues/15661

**Comment by lzap** [comment]
Fixed in 1.1.0 update and let me know.

**Comment by lekemula** [comment]
Seems to be fixed now, thank you! ❤️
